Dockerfile

```diff
@@ -29,6 +29,8 @@ COPY --from=app-stage ldes-server-port-ingest-rest/target/ldes-server-port-inges
 COPY --from=app-stage ldes-server-port-ingest/target/ldes-server-port-ingest-jar-with-dependencies.jar ./lib/
 COPY --from=app-stage ldes-server-port-fetch/target/ldes-server-port-fetch-jar-with-dependencies.jar ./lib/
 COPY --from=app-stage ldes-server-port-fetch-rest/target/ldes-server-port-fetch-rest-jar-with-dependencies.jar ./lib/
+COPY --from=app-stage ldes-server-port-fetch-application-layer/target/ldes-server-port-fetch-application-layer-jar-with-dependencies.jar ./lib/
```

Comment:
I don't really understand why you added this new module.
Can't you use spring modulith and only activate the fetch profile and make sure that works? Then you don't have to update the entire server.